plugins/scaffolder-backend/api-report.md

```diff
@@ -187,7 +188,7 @@ export function createGithubRepoCreateAction(options: {
   allowMergeCommit?: boolean | undefined;
   requireCodeOwnerReviews?: boolean | undefined;
   requiredStatusCheckContexts?: string[] | undefined;
-  repoVisibility?: 'internal' | 'private' | 'public' | undefined;
+  repoVisibility?: 'public' | 'private' | 'internal' | undefined;
```

Comment:
All of these similar changes were previously in alphabetical order. Why the change?

Reply:
I noticed this too. This is the result of running `yarn build:api-reports:only plugins/scaffolder-backend`. But I guess something is still off, because the tests on this file are still failing. Not sure what it is yet.

I ran `yarn build:api-reports:only plugins/scaffolder-backend` because running `yarn build:api-reports:only` results in this error:

```
# Generating package API reports
## Processing plugins\user-settings
Error: The API Report for plugins\user-settings is not allowed to have warnings
```

Reply:
Hmm, will have to look into this separately, but for now you'll need to run the full `yarn build:api-reports`

Reply:
I keep struggling with the `build:api-reports` command. It keeps giving me errors on other plugins that are not allowed to have warnings. Any clues?

```
$ tsc --project C:\DEV\backstage-original\tsconfig.json --skipLibCheck false --incremental false
# Generating package API reports
## Processing plugins\xcmetrics
Error: The API Report for plugins\xcmetrics is not allowed to have warnings
    at runApiExtraction (C:\DEV\backstage-original\scripts\api-extractor.ts:562:13)
    at async main (C:\DEV\backstage-original\scripts\api-extractor.ts:1360:5)
```

Reply:
It takes a really long time to run. Perhaps this is due to a timeout?

Anyway. I ran `yarn build:api-reports plugins/scaffolder-backend` again, because that does give results. It reverted some of the earlier changes, but still fails the test unfortunately. I think because the order of one property is still changed in this PR.

Reply:
Alright, dunno what might by up tbh, gonna verify that master isn't broken. I'll ship a fix to your PR in a bit too

Reply:
Tried building API reports on latest master and it seems all good. Most likely something with your local environment that's up. You've ran `yarn install` as well right?

Either way I pushed up an updated API report so hopefully should be good to go now

Reply:
Thanks! I did indeed `yarn install` and then `yarn build:api-reports`. Not sure what it is. But happy that you fixed it! 👍